dune-project

```diff
@@ -19,9 +19,15 @@
  (synopsis "ReScript compiler")
  (depends
   (ocaml
-   (>= 4.10))
+   (>= 5.2.0))
```

Comment:
4.13 is the oldest version that still works, and CI actually verifies that it works.